cli-plugins/manager/manager.go

```diff
@@ -74,9 +76,17 @@ func addPluginCandidatesFromDir(res map[string][]string, d string) {
 		return
 	}
 	for _, dentry := range dentries {
-		switch dentry.Type() & os.ModeType { //nolint:exhaustive,nolintlint // no need to include all possible file-modes in this list
-		case 0, os.ModeSymlink:
-			// Regular file or symlink, keep going
+		switch mode := dentry.Type() & os.ModeType; mode { //nolint:exhaustive,nolintlint // no need to include all possible file-modes in this list
+		case os.ModeSymlink:
+			if _, err := os.Stat(filepath.Join(d, dentry.Name())); err != nil {
+				// Skip broken symlinks unless debug is enabled. With debug
+				// enabled, this will  print a warning in "docker info".
+				if errors.Is(err, os.ErrNotExist) && !debug.IsEnabled() {
+					continue
+				}
+			}
```

Comment:
@Benehiko @vvoland I updated this to ignore the broken symlink by default, but with `--debug` enabled, it returns to the old behavior, which will produce an error that's printed as a warning